content_schemas/allowed_document_types.yml

```diff
@@ -10,7 +10,6 @@
 - authored_article
 - board_member_role
 - business_finance_support_scheme
-- calculator
```

Comment:
Can the corresponding `.jsonnet` (i.e. https://github.com/alphagov/publishing-api/blob/main/content_schemas/formats/calendar.jsonnet) also be removed in this commit?

Reply:
I think this comment also applies to the other content types that are being removed, but I might be missing some context!

Reply:
Yup, I've tried to remove all the corresponding jsonnet files with them. I think you've just made a minor error with this catch as you've pulled out the calendar.jsonnet file not calculator.jsonnet.

Thanks for taking a look.